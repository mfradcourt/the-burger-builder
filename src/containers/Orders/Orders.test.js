import React from 'react';
import { shallow } from "enzyme";

import { Orders } from './Orders';

describe('<Orders />', () => {
  let wrapper;

  it('should render a default Orders', () => {
    wrapper = shallow(
      <Orders/>
    );

    expect(wrapper).toMatchSnapshot();
  });

  it('should render Orders when there are orders', () => {
    const orders = [
      {
        id: 1,
        ingredients: {
          meat: 2,
          salad: 5
        },
        price: 10
      },
      {
        id: 2,
        ingredients: {
          meat: 6,
          salad: 5
        },
        price: 25
      }
    ];

    wrapper = shallow(
      <Orders/>
    );

    wrapper.setState({orders: orders});
    expect(wrapper).toMatchSnapshot();
  });
});
