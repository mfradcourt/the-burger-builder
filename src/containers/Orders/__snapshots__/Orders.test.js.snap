// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Orders /> should render Orders when there are orders 1`] = `
<div>
  <order
    ingredients={
      Object {
        "meat": 2,
        "salad": 5,
      }
    }
    key="1"
    price={10}
  />
  <order
    ingredients={
      Object {
        "meat": 6,
        "salad": 5,
      }
    }
    key="2"
    price={25}
  />
</div>
`;

exports[`<Orders /> should render a default Orders 1`] = `<div />`;
